src/pages/Onboard/NameTeam.vue

```diff
@@ -130,18 +136,92 @@ export default {
       await this.createLicense()
       if (!this.updateServerError) this.goToResources()
     },
-    goToResources() {
+    async accept(pt) {
+      try {
+        this.accepting = true
+        const invitationId = pt.id
+        const accepted = await this.acceptMembershipInvitation(invitationId)
+        this.redirectTenant = pt.tenant.slug
+        if (accepted.accept_membership_invitation.id) {
+          this.redirectTenant = pt.tenant.slug
+          this.accepting = false
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
+      this.loading = false
```

Comment:
I'm not sure where loading is being set elsewhere that would require it being set again here. Should this, `accepting`, and `deleting` be combined into a single `loading` prop?

Reply:
tried it, but I ended up deciding to keep them separate. I think they should be used a bit more consistently now though!